api/src/models/contragent.py

```diff
@@ -0,0 +1,17 @@
+from sqlalchemy.orm import Mapped, mapped_column
+from sqlalchemy import String, Float
+
+from ..database import Base
+from . import core_model
```

Comment:
from src.models.core_model import CoreModel

Reply:
Deleted due to unused rn